packages/redux/src/contents/selectors.ts

```diff
@@ -321,3 +327,117 @@ export const getSEOMetadataResult = (
  */
 export const getContent = <T>(state: StoreState, hash: string) =>
   getEntityById(state, 'contents', hash) as ContentsEntity<T> | undefined;
+
+/**
+ * Returns the error thrown to the getSEOFiles request.
+ *
+ * @example
+ * ```
+ * import { getSEOFilesError } from '@farfetch/blackout-redux/contents';
```

Comment:
```suggestion
 * import { getSEOFilesError } from '@farfetch/blackout-redux';
```

The same for the remaining selectors.

Reply:
But the selectors are inserted in the `content` folder

Reply:
I know but starting with v2 we want to emphasize imports from the root of the package instead of folder imports. All other jsdocs with import examples use that (except one for analytics but that is a special case).

Reply:
Resolved! ✅